Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from /tg/station at https://github.com/tgstation/tgstation/commit/6bfe3b2e4fcbcdac9159dc4f0327a82ddf05ba7bi. Modified by Radezolid for random intercom spawner",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+	{
+	    "name": "random_intercom",
+        "directions": 4
```

Comment:
indent is sus on this line

Reply:
It was a stupid error, tab vs 4 spaces. Didn't notice it at the time. Fixed.

Reply:
Done